api/valor_api/backend/core/label.py

```diff
@@ -320,6 +320,80 @@ def get_labels(
     }
 
 
+def get_paginated_labels(
```

Comment:
I made this a new function since `get_labels` is used both for the end point and for a bunch of internal calls. The headers are only useful for the endpoints, so it makes sense to split `get_labels` from `get_paginated_labels`.

Reply:
I would prefer that API endpoint related code (like pagination) be implemented in CRUD rather than the backend.

This would work by augmenting the getters in `_read.py` which are meant to be the interface between the API and the backend.

Reply:
responded via slack. we want to use offset and limit in psql for performance reasons, and don't want to make db calls in crud functions